Changed search results page

jrodonohue opened the pull request:
The search results page has been changed to display a variable number of books. Tapping a search result will open the result in the user's browser. The search by cover button does not navigate to a default review page, but it does nothing. More work needs to be done to display reviews in the app, but the results page should be good enough.

mtuthill reviewed (commented):
Looks great! What do you need done for the scrapers? 
The only thing I'd suggest is to just delete the app/src/main/java/com/example/byitscover/review_list_page/ReviewListPage.java file since it's empty

jrodonohue commented:
> Looks great! What do you need done for the scrapers?
> The only thing I'd suggest is to just delete the app/src/main/java/com/example/byitscover/review_list_page/ReviewListPage.java file since it's empty

In the Storybook scraper I replaced `0.0` with `null` on line 83, and on line 72, I replaced `google.com` with `https://google.com` because an exception was thrown due to the string omitting a protocol.

mtuthill commented:
> > Looks great! What do you need done for the scrapers?
> > The only thing I'd suggest is to just delete the app/src/main/java/com/example/byitscover/review_list_page/ReviewListPage.java file since it's empty
> 
> In the Storybook scraper I replaced `0.0` with `null` on line 83, and on line 72, I replaced `google.com` with `https://google.com` because an exception was thrown due to the string omitting a protocol.

Got it, wasn't sure if you had identified additional things that needed to be changed
